views/backend/postback.php

```diff
@@ -19,31 +17,35 @@
         </div>
     <div class="container">
 
-    <?php
-        if(isset($_POST['delete'])){
-            $article = $model_article->delete_article();
-            $comment = $model_comment->delete_article_comments();
+<?php
+   function form_page_postback(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [form_page_postback accesses the super-global variable $_FILES.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)